tests/test_web_llm_runtime_config.py

```
@@ -801,112 +801,85 @@ def test_game_llm_config_uses_advanced_model_for_api_reasoning_capability(monkey
+        _assert_hud(game.state_payload())
+        assert not db_path.exists() or db_path.read_bytes() != b"existing-progress"
+        assert calls == [], f"fresh 构造不应触发 LLM 调用，实得 {calls}"
+    finally:
+        try:
+            game.session.close()
+        except Exception:
+            pass
+
+
+def test_continue_load_save_reset_reach_hud_zero_llm_calls(tmp_path, monkeypatch):
+    """#1228 行为：continue / load_save / 重置进 HUD，全程零 LLM 调用直至真实动作。"""
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF002)